### ISSUE TITLE
Ability to setup cron for dumps and restore from a dump when setting it up

### ISSUE THREAD

**Issue by AbdealiJK** [UnderGreen/ansible-role-mongodb#104]
Would be great if we could have a variable to setup a cron to keep dumping the mongodb to a specific location and also a variable to restore the mongodb from one of those snapshots

**Comment by jebrage** [comment]
Yes. A task to load the DB from a dump at start up would be great too.
